packages/composables/package.json

```diff
@@ -15,8 +15,8 @@
     "update:update": "ncu -u"
   },
   "dependencies": {
-    "@vue-storefront/vendure-api": "1.2.0",
-    "@vue-storefront/core": "~2.4.6"
+    "@vue-storefront/core": "~2.4.6",
+    "@vue-storefront/vendure-api": "link:../api-client"
```

Comment:
You do not need this link. But if you do, please remember not to push it to the code repo ;)